### ISSUE TITLE
Data for tutorial no longer available

### ISSUE THREAD

**Issue by MC-Alejandro** [bulik/ldsc#282]
Hi.
I was trying to wget the data you mention on the heritability and genetic correlation tutorial but the location specified cannot be found.
I was wondering if you could respecify the path to those files.

(ldsc) root@13b3dc37dc2f:/data/data_totorial# ls
1kg_eur  eur_w_ld_chr.tar.bz2  w_hm3.snplist.bz2
(ldsc) root@13b3dc37dc2f:/data/data_totorial# wget www.med.unc.edu/pgc/files/resultfiles/pgc.cross.bip.zip
--2021-03-16 07:33:22--  http://www.med.unc.edu/pgc/files/resultfiles/pgc.cross.bip.zip
Resolving www.med.unc.edu (www.med.unc.edu)... 152.19.9.36
Connecting to www.med.unc.edu (www.med.unc.edu)|152.19.9.36|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-03-16 07:33:22 ERROR 404: Not Found.

Many thanks!

**Comment by tobiaswolfram** [comment]
They are also available here: https://gist.github.com/bulik/c194b929a340ed3ff610

**Comment by sjdavenport** [comment]
I don't think this works anymore, at least I can't find these files at that link.
